### PR TITLE
Feature: find rom in zip

### PR DESCRIPTION
**Pull request by maxim-zhao**
Iterates through the zip looking for a file with a known extension. This does not preserve compatibility with a zip containing no known extensions (would previously load the first file as SMS, now loads the last) as I didn't see the point.

### CODE REVIEW

**Comment by maxim-zhao** [comment]
Too many commits :angry: I'll figure it out...

**Comment by ocornut** [comment]
Everyone does this mistake, git is weird.
PR = tied to one branch
Official git way is to branch the hell out every time you breath (and maybe have a local branch with everything merged..) i can't understand/agree with this way of making things frankly. or, you can create the pr branch after the fact by cherry picking commits from your working branch.

File: meka/srcs/file.c [hunk 1]
```diff
@@ -232,7 +232,7 @@ bool    Load_ROM(t_load_mode load_mode, bool user_verbose)
         case LOAD_MODE_COMMANDLINE:
             Quit_Msg("%s\n\"%s\"\n", meka_strerror(), g_env.Paths.MediaImageFile);
             // Quit_Msg(meka_strerror());
-            return false;
+//            return false;
```
Comment (ocornut) [inline meka/srcs/file.c]: Why that comment?

**Comment by ocornut** [comment]
I'm trying to figure out how to cherry pick your commit from here..

**Comment by ocornut** [comment]
Merged, ahah!

File: meka/srcs/file.c (hunk 1 above)
Comment (maxim-zhao) [inline meka/srcs/file.c]: Mistake, I turned on /w4 but didn't intend to PR it. (Unreachable code due to exit(1) in Quit_Msg().)

**Pull request closed without merging**